src/tours/Tour/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react';
+import Tour from 'reactour';
+import {
+    getGroupComparisonSteps,
+    getVirtualStudySteps,
+    groupComparisonId,
+    virtualStudyId,
+} from '../Steps';
+import { TourProps, TourMapProps } from './types';
+import './styles.scss';
+
+export const setTourLocalStorage = (id: string, value: string) => {
+    localStorage.setItem('web-tour', id);
```

Comment:
use constant for "web-tour" so that we can use it again for retrieval down below.